site/docs/reference/Connectors/capture-connectors/README.md

```diff
@@ -0,0 +1,29 @@
+# Capture connectors
+
+Estuary's available capture connectors are listed in this section. Each connector has a unique configuration you must follow in your [catalog specification](concepts/catalog-entities/README.md); these will be linked below the connector name.
+
+:::info Beta
+More configurations coming to the docs soon. [Contact the team](mailto:info@estuary.dev) for more information on missing connectors.
+:::
+
+Links to the package page on GitHub are also provided, including the most recent Docker image, which you'll need for certain configuration methods.
+
+Estuary is actively developing new connectors, so check back regularly for the latest additions. We’re prioritizing the development of high-scale technological systems, as well as client needs.
+
+## Available capture connectors
+
+* Amazon Kinesis
+  * Configuration
+  * [Package ](https://github.com/estuary/connectors/pkgs/container/source-kinesis)
+* Amazon S3
+  * Configuration
+  * [Package ](https://github.com/estuary/connectors/pkgs/container/source-s3)
+* Apache Kafka
+  * Configuration
+  * [Package ](https://github.com/estuary/connectors/pkgs/container/source-kafka)
+* Google Cloud Storage
+  * Configuration
+  * [Package ](https://github.com/estuary/connectors/pkgs/container/source-gcs)
+* PostgreSQL
+  * [Configuration](./PostgreSQL.md)
+  * [Package](https://github.com/estuary/connectors/pkgs/container/source-postgres)
```

Comment:
Perhaps include the Docker image URL on this page, maybe something like `Package (ghcr.io/estuary/source-postgres:dev)` (and similarly for other connectors obviously)?

That way the user doesn't have to follow a link just to get a copy-pastable connector image URL, since aside from having that URL the GitHub packages web page isn't likely to be very helpful to most users.